app/views/incomeSources/manage/ConfirmReportingMethod.scala.html

```diff
@@ -18,32 +18,45 @@
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 @import forms.incomeSources.manage.ConfirmReportingMethodForm
 @import views.html.helpers.injected.ErrorSummary
+@import views.html.components.h1
 
-@this(
-        mainTemplate: unifiedLayout,
-        govukCheckboxes: GovukCheckboxes,
-        govukButton: GovukButton,
-        formWithCSRF: FormWithCSRF,
-        errorSummary: ErrorSummary
+@this(mainTemplate: unifiedLayout,
+    govukCheckboxes: GovukCheckboxes,
+    govukButton: GovukButton,
+    formWithCSRF: FormWithCSRF,
+    errorSummary: ErrorSummary,
+    govukWarningText : GovukWarningText,
+    h1: h1
 )
 
-@(
-        form: Form[ConfirmReportingMethodForm],
-        backUrl: String,
-        postAction: Call,
-        isAgent: Boolean,
-        taxYearStartYear: String,
-        taxYearEndYear: String,
-        reportingMethod: String
-)(implicit messages: Messages, user: auth.MtdItUser[_])
+@(form: Form[ConfirmReportingMethodForm],
+    backUrl: String,
+    postAction: Call,
+    isAgent: Boolean,
+    taxYearStartYear: String,
+    taxYearEndYear: String,
+    newReportingMethod: String,
+    isCurrentTaxYear: Boolean)(implicit messages: Messages, user: auth.MtdItUser[_])
 
 @prefix = @{ "incomeSources.manage.propertyReportingMethod" }
 
 @getMessage(key: String, args: String*) = @{
     messages(s"$prefix.$key", args: _*)
 }
 
-@mainTemplate(pageTitle = getMessage(s"heading.$reportingMethod"),
+@warningMessage(isAgent: Boolean) = @{
+    if(isAgent) {
+        messages("incomeSources.manage.propertyReportingMethod.warning.agent")
+    } else {
+        messages("incomeSources.manage.propertyReportingMethod.warning.individual")
+    }
+}
+
+@showWarningText = @{
```

Comment:
not seeing warning when changing from annual to quarterly

Reply:
Fixed in latest commit